### ISSUE TITLE
UPDS patch fails on 5.10 

### ISSUE THREAD

**Issue by B3HOID** [Frogging-Family/linux-tkg#775]
When I try to select UPDS in the script menu and try to compile the Linux 5.10 kernel, I get this error during the compile process:

```
kernel/sched/pds.c:41:10: fatal error: ../../fs/io-wq.h: No such file or directory
   41 | #include "../../fs/io-wq.h"
```
This is weird because I looked under /fs and there's no io-wq.h file. There is however one under io_uring. 

Where's the proper header file?

**Comment by B3HOID** [comment]
Ok I fixed that error by changing the proper header path to "../../io_uring/io-wq.h"

But now I get new errors: 
```

kernel/sysctl.c:2531:13: error: ‘rr_interval’ undeclared here (not in a function)
 2531 |   .data  = &rr_interval,
      |             ^~~~~~~~~~~
kernel/sysctl.c:2536:15: error: ‘one_thousand’ undeclared here (not in a function); did you mean ‘ten_thousand’?
 2536 |   .extra2  = &one_thousand,
      |               ^~~~~~~~~~~~
      |               ten_thousand
kernel/sysctl.c:2540:13: error: ‘sched_yield_type’ undeclared here (not in a function)
 2540 |   .data  = &sched_yield_type,
      |             ^~~~~~~~~~~~~~~~
kernel/sysctl.c:2545:15: error: ‘two’ undeclared here (not in a function)
 2545 |   .extra2  = &two
```
What's wrong here

**Comment by Tk-Glitch** [comment]
Looks like 5.10 got some merges from newer releases. Those errors are ringing a bell, I'll check that out asap.

**Comment by Tk-Glitch** [comment]
All those dead and unmaintained schedulers are using similar mechanisms which got changed upstream, so nothing surprising here.

**Comment by B3HOID** [comment]
Ok I think I think I fixed it by changing the sysctl.c file manually.

**Comment by Tk-Glitch** [comment]
Not an actual fix for current 5.10, but the target was changed to an earlier 5.10 revision which works since we do not plan to maintain that anymore: https://github.com/Frogging-Family/linux-tkg/commit/f8438aab4edf1cf0e49dd63e159cf0fe84926059
